components/tree/demo/customized-icon.ts

```diff
@@ -5,7 +5,16 @@ import { Component, OnInit } from '@angular/core';
   template: `
     <nz-tree
       [nzData]="nodes"
-      nzShowIcon="true">
+      nzShowIcon="true"
+      [nzExpandedIcon]="'down'">
+    </nz-tree>
+    <nz-tree
+      [nzData]="nodes"
+      nzShowIcon="true"
+      [nzExpandedIcon]='expandedIcon'>
+      <ng-template #expandedIcon let-node>
+        <i nz-icon [type]="node.origin.icon" class="ant-tree-switcher-icon"></i>
```

Comment:
nzExpandedIcon supports TemplateRef only is better, remove string type, and should consider the nzShowLine case.